app/script/entity/message/Message.js

```diff
@@ -261,7 +264,7 @@ z.entity.Message = class Message {
 
   // Start the ephemeral timer for the message.
   startMessageTimer() {
-    if (this.ephemeral_timeout_id) {
+    if (this.messageTimerStarted) {
```

Comment:
Looks like we can remove the timeout ID in the constructor and in it the started check.